client/swagger/swagger.go

```diff
@@ -0,0 +1,12 @@
+// Package swagger contains the client library code generated automatically from the swagger definitions.
+// The code present in http, models and operations must not be modified.
+// To run the source generation, swagger binary must be present in PATH.
+// To install the swagger binary, run:
+//
+//	go install github.com/go-swagger/go-swagger/cmd/swagger@latest
+//
+// For more information, see: https://github.com/go-swagger/go-swagger
+package swagger
+
+//go:generate swagger generate client -f ../../docs/swagger/swagger.json -t ../../ -c client/swagger/http -m client/swagger/models -a /client/swagger/operations
```

Comment:
Can you try
```
go:generate go run github.com/go-swagger/go-swagger/cmd/swagger@latest generate client -f ../../docs/swagger/swagger.json -t ../../ -c client/swagger/http -m client/swagger/models -a /client/swagger/operations
```
If it works, then we don't need the additional step in the workflow
If it does not work, at least use a fixed version instead of `latest` since the generated code may change when version change and may break CI and dependants

Reply:
I think it is cached in ~/go/pkg, at least it is fast for the 2nd time on my machine, as long as you use a fixed version